lib/rules/lighter-image-react.js

```diff
@@ -0,0 +1,71 @@
+/**
+* @fileoverview Rule to avoid heavy image file formats
+* @author Martyna Babiak
+*/
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const moduleVisitor = require("eslint-module-utils/moduleVisitor");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/** @type {import('../shared/types').Rule} */
+module.exports = {
+    meta: {
+        type: "problem",
+
+        docs: {
+            description: "Avoid heavy formats of image files",
+            category: "Code improvements",
+            recommended: false,
+            url: ""
+        },
+        messages: {
+          suggestion: "Format of image file can be changed to WebP, SVG, GIF or JPG. CO2 reduction: up to 99% of the image file."
```

Comment:
can we please change it to the same error log as in ec0lint-style? https://github.com/ec0lint/ec0lint-style/blob/main/lib/rules/lighter-image-files/index.js